### PR TITLE
content updates

### PR DESCRIPTION
**Pull request by pmgrove**
:v:

/cc @zendesk/vegemite 

### Description

Update en.json file with marketplace listing content

### References
* JIRA: 

### Risks
* [low] - app content may not show on marketplace

### CODE REVIEW

File: src/translations/en.yml [hunk 1]
```diff
@@ -6,10 +6,36 @@ packages:
...
+
+By enabling this app, You agree to the [Built by Zendesk Terms of Use](https://www.zendesk.com/company/built-by-zendesk-agmt/)."
+  - translation:
+      key: "txt.apps.related_tickets_app.app.installation_instructions"
+      title: "Installation Instructions for App: Show Related Tickets"
+      value: "To install the Show Related Tickets app:
+
+1. Sign in to your Zendesk Support account, then click the Admin icon in the sidebar.
+
+2. Select Apps > Marketplace, then find the Show Related Tickets app and click the tile.
```
Comment (svizzari) [inline src/translations/en.yml]: Were you planning on removing steps 1 & 2 from this list?

Comment (pmgrove) [inline src/translations/en.yml]: We could remove all install instructions for this app but that might look weird so I would say we leave as is.

**Comment by pmgrove** [comment]
@zendesk/i18n

**Comment by mariasica** [comment]
👍

**Pull request merged**